Beam/resources/assets/js/remplib.js

```diff
@@ -48,6 +48,16 @@ remplib = typeof(remplib) === 'undefined' ? {} : remplib;
 
         timeSpentActive: false,
 
+        trackedProgress: [],
+
+        trackedArticle: null,
+
+        trackedProgressInterval: 5,
+
+        trackedProgressPath: this.config.tracker.readingProgress.target,
```

Comment:
This target shouldn't be customizable. Tracker's URL is customizable, the path should be hardcoded as in all other `/track/*` calls.